src/Sylius/Behat/Page/Admin/Product/ShowPage.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Product;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class ShowPage extends SymfonyPage implements ShowPageInterface
```

Comment:
Thats true, but shouldn't it be misleading with bigger amount of pages? But we can skip it for now